Dockerfile.ubi

```diff
@@ -3,16 +3,37 @@ ARG BASE_IMAGE
 
 FROM --platform=${BUILDPLATFORM} ${BUILDER_IMAGE} as SRC
 
-FROM ${BASE_IMAGE:-registry.access.redhat.com/ubi8/ubi-minimal:latest}
+FROM ${BASE_IMAGE:-registry.access.redhat.com/ubi8/ubi:latest} as minimal-ubi
```

Comment:
any reason of using ubi image instead of ubi-minimal but still tagging it as ubi-minimal?

Reply:
@IdanAdar @bainsy88 ^

Reply:
The reason the top base is UBI and not ubi-minimal is because some of the scripts we use to for slimming down the image are not in ubi-minimal. No issue with changing the name from minimal-ubi for clarity